src/refresh.cc

```diff
@@ -480,11 +480,14 @@ refreshCheck(const StoreEntry * entry, HttpRequest * request, time_t delta)
         }
 
 #endif
+        debugs(22, 3, "returning STALE_EXPIRES");
         return STALE_EXPIRES;
     }
 
-    if (sf.max)
+    if (sf.max) {
+        debugs(22, 3, "returning STALE_MAX_RULE");
```

Comment:
Some existing debugs() in this function already use this (correct) minimalist style to report outcomes.

Some debugs() also add "YES/NO prefixes that are rather misleading for a non-boolean function that is not called something like isFresh(). Some debugs() also try to explain why a particular outcome was reached by describing the last condition checked, but such reporting can easily mislead because all but the very first outcome depend on _multiple_ preconditions. For the two `return` statements reported in this PR, just letting the developer know which exit path got triggered is the best strategy IMO.